lib/spack/docs/getting_started.rst

```diff
@@ -23,6 +23,7 @@ before Spack is run:
    executables for extracting source code
 #. The ``patch`` command to apply patches
 #. The ``git`` and ``curl`` commands for fetching
+#. If using CVS, the ``cvs`` command for fetching
```

Comment:
We don't specifically call out `go` or `hg`, so I don't think we need to call out `cvs` here either.

Other than this, the PR looks good to me (and I will approve it with that change).

Reply:
True. Done.